docs/modules/general/proto.mdx

```diff
@@ -42,10 +42,40 @@ Mapmakers should always use the latest supported proto version, and this may be
 ### Changes in 1.5.0
 
 :::note
-`1.5.0` is a upcoming protocol version that is currently in its planning stage and therefore, **not implemented**.
-If you would like to learn more or propose changes, please visit [PGM#1267 on the issue tracker](https://github.com/PGMDev/PGM/issues/1267).
+`1.5.0` is now available. As the latest protocol version, it is not fully polished, so bugs are expected.
+If you would like to learn more or report issues, please visit the [PGM issue tracker](https://github.com/PGMDev/PGM/issues).
 :::
 
+#### Breaking
+
+- Filters and variables are now their own IDs that will be reserved by PGM. Those IDs can be referenced anywhere in the XML.
+  - This applies to the following filters: `observing`, `participating`, `alive`, `dead`, `match-idle`, `match-starting`, `match-running`, `match-finished`, `match-started`, `crouching`, `walking`, `sprinting`, `grounded`, `flying`, `can-fly`, and `void`.
+  
+    ```xml
+    <!-- Sample proto 1.4.2 XML -->
+    <filters>
+        <alive id="alive"/> <!-- id="alive" is not necessary in proto 1.5.0 -->
+    </filters>
```

Comment:
```suggestion
    <!-- Sample proto 1.4.2 XML -->
    <filters>
        <alive id="alive"/> 
    </filters>
    <some-other-feature filter="alive"/>
    
    <!-- Sample proto 1.5.0 XML -->
    <filters>
      <!-- Defining the alive id is not necessary, as it's a built-in now -->
    </filters>
    <some-other-feature filter="alive"/>    
```
What's not necessary isn't the `id="alive"`, but rather the whole defining of that filter in the first place, in order for it to be usable in other features